src/main/java/io/antmedia/muxer/Muxer.java

```diff
@@ -582,6 +584,7 @@ public String getExtendedName(String name, int resolution, int bitrate, String f
 		if (addDateTimeToResourceName) 
 		{
 			LocalDateTime ldt =  LocalDateTime.now();
+			currentVoDTimeStamp = System.currentTimeMillis();
```

Comment:
There is already a LocalDateTime object one line above. No need to get the `System.currentTimeMillis();` again. Please use just one single date time object because we rarely encounter differences between ldt and currentVoDTimeStamp again.